lago/cmd.py

```diff
@@ -83,6 +83,13 @@
 @lago.plugins.cli.cli_plugin_add_argument(
     '--template-store',
     help='Location to store templates at',
+    default='/var/lib/lago/store',
```

Comment:
looks like [argparse supports](https://docs.python.org/3/library/argparse.html#help) interpolation only for the ``help`` keyword, the INI part should be easier as ``configparser`` supports interpolation. Either way to make generic support would involve some in-house solution.
If we change it to base_dir, wouldn't that be too restrective on the users? (having store, repos, subnets directories in one place, while one might want to move store or repos to a different device with more space for instance )